.github/ISSUE_TEMPLATE/iceberg_bug_report.yml

```diff
@@ -0,0 +1,93 @@
+---
+name: Iceberg Bug report 🐞
+description: Problems, bugs and issues with Apache Iceberg
+labels: ["kind:bug"]
+body:
+  - type: markdown
+    attributes:
+      value: "
+        <img src='https://iceberg.apache.org/docs/latest/img/Iceberg-logo.png' align='center'>
+        Thank you for finding the time to report the problem!
+
+        We really appreciate the community's efforts to improve Iceberg."
+  - type: dropdown
+    attributes:
+      label: Apache Iceberg version
+      description: >
+        What Apache Iceberg version are you using?
+      multiple: false
+      options:
+        - "0.13.1 (latest released)"
+        - "0.13.0"
```

Comment:
One thing that comes up _all the time_ with Spark (and similarly with Flink, but less so), is that we updated the runtime JAR names to be by version, e.g. `iceberg-spark-runtime-3.2_2.12:0.13.1`.

However, we left Spark 3.0 as the legacy name `iceberg-spark3-runtime_2.12:0.13.1`. Can we possibly ask which jar they are using?

I have wanted to _stop_ publishing `iceberg-spark3-runtime` entirely, and change that also to `iceberg-spark-runtime-3.0_2.12:0.14.0`, but a few people have mentioned that as "breaking". I personally don't see that as a problem, as the entire change is technically breaking (moving to the per version JAR release).

**TLDR** - In any case, because we use version specific jars, I feel it's appropriate to ask users which jar they're using (even if we do make the full change - though with the update it will likely be _somewhat less_ of an issue).

Reply:
I've added another field to define the version of the Query Engine. Would that be sufficient?

Reply:
Yeah that's definitely sufficient 👍